### PR TITLE
Lime 166 dva detail entry page

### PR DESCRIPTION
**Pull request by chinmayi-gds**
This has the automation tests for the following tickets :

DVA Detail Entry Page  https://govukverify.atlassian.net/browse/LIME-166
DVA Retry Screen  https://govukverify.atlassian.net/browse/LIME-147

Also, have done refactored the DVLA tests to remove duplicates.

### CODE REVIEW

File: src/test/java/gov/di_ipv_fraud/pages/EnterYourDetailsExactlyDVLAPage.java [hunk 1]
```diff
@@ -42,9 +40,6 @@ public EnterYourDetailsExactlyDVLAPage() {
     @FindBy(id = "expiryDate-year")
     public WebElement LicenceValidToYear;
 
-    @FindBy(id = "licenceIssuerRadio")
-    public WebElement DVLA;
-
     @FindBy(id = "licenceIssuerRadio-DVLA-label")
     public WebElement DVLALabel;
```
Comment (ElliotMurphyGDS) [inline src/test/java/gov/di_ipv_fraud/pages/EnterYourDetailsExactlyDVLAPage.java:44]: Are we now only finding the label? Why Are we not finding the actually radio?

**Review by ElliotMurphyGDS — APPROVED** [review]

File: src/test/java/gov/di_ipv_fraud/pages/EnterYourDetailsExactlyDVLAPage.java (hunk 1 above)
Comment (chinmayi-gds) [reply src/test/java/gov/di_ipv_fraud/pages/EnterYourDetailsExactlyDVLAPage.java:44]: @ElliotMurphyGDS : We are using the actual radio. I removed the above lines because this element was already available here https://github.com/alphagov/di-ipv-cri-fraud-test/blob/main/src/test/java/gov/di_ipv_fraud/pages/DrivingLicencePageObject.java
under this name 
    @FindBy(id = "licenceIssuerRadio")
    public WebElement radioBtnDVLA;
    
    So just reused that and removed the duplicate.
